src/web/routes/application/select-address/select-address.js

```diff
@@ -17,22 +19,33 @@ const buildAddressOption = result => ({
   text: result.ADDRESS
 })
 
-const behaviourForGet = () => async (req, res, next) => {
+const behaviourForGet = () => (req, res, next) => {
   res.locals.addresses = req.session.postcodeLookupResults.map(buildAddressOption)
   // Manual address is further in the flow than select-address, therefore this line is needed to prevent the state machine from redirecting the user back to select-address.
   req.session.nextAllowedStep = '/manual-address'
```

Comment:
`req.session.nextAllowedStep` happens in a few files .. do you think this should be abstracted into a function? (probably not for this PR)

Reply:
For such a simply assignment operation I'm not sure it warrants it's own function.